vars/collectBuilderLogs.groovy

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+def call(){
+    sh """#!/bin/bash
+
+	set -xe
+
+	SHORT_SCT_TEST_ID=\$(echo \$SCT_TEST_ID | cut -c1-8)
+    sudo journalctl --no-tail --no-pager -o short-precise > builder-\$SHORT_SCT_TEST_ID.log
```

Comment:
maybe adding  --since "x hour ago"

Reply:
why not all ?

Reply:
if builder will be running for couple days, it will be big and redundant

Reply:
it would be very small compared to SCT and DB logs.

we don't yet know exactly what we need of it, I would rather first collect it all.

Reply:
FY, test run for 2h, the uncompressed log was 485K